cmd/clusterctl/examples/ibmcloud/cluster.yaml.template

```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
     clusterNetwork:
         services:
-            cidrBlocks: ["10.96.0.0/12"]
```

Comment:
@morvencao can we set the ip CIDR for VMs provisioned by IBM Cloud?

Reply:
It should be. And the issue https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/issues/153 should be caused CIDR overlap. So maybe this is a temporary fix.

Maybe @xunpan @jichenjc have more info about VM CIDR.

Reply:
My concern is if end user configure the VM CIDR start from 20, calico will crash again.

@morvencao Can you help test if your fix works? If it works, then we need to update this PR to let VM CIDR configurable.

Reply:
@gyliu513 Change default service CIDR works in my env.

Reply:
OK, then I suggest we update this PR by enabling end user can set the CIDR for the VMs created in IBM Cloud.

Reply:
I have some comments here:
https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/issues/153

IBM cloud default private ip seems to be 10.x.x.x/8 which may leads to the issue

we can't set 20.x as it's not a private IP so we can only use this 20.x as example and hint to find the issue
but not something we can recommend in our solution 
let's switch to that issue for further discussion @morvencao @gyliu513